composer.json

```diff
@@ -30,6 +30,7 @@
     },
     "config": {
         "allow-plugins": {
+            "cweagans/composer-patches": true,
```

Comment:
we don't need this line and should remove it to avoid confusion

Reply:
sorry, yes I intended to remove that line.